Documentation/Generalities.md

```diff
@@ -0,0 +1,33 @@
+# Generalities #
+
+### BlackScreen ###
+Show or remove black screen. Also removes projectiles from the combat zone
+
+* BooleanEnables
+
+### Sound ###
+Play any of the sounds (e.g. "Flash")
+
+* SoundName
+
+### Music ###
+Play music. Currently, only "mus_zz_megalovania" works
+
+* MusicName
+
+### TLPause ###
+Pause the timeline
+
+* NO ARGUMENTS
+
+### TLResume ###
+Resume the timeline. Useful when "scheduled" by some other action (like CombatZoneResize)
+
+* NO ARGUMENTS
+
+### EndAttack ###
+Ends the attack allowing the game to continue.
+This must be called, or the attack will never end.
+
+* NO ARGUMENTS
+* 
```

Comment:
Extra bullet

Reply:
What do your mean with "extra bullet"?

Reply:
This extra bit here:
![image](https://user-images.githubusercontent.com/975426/82857497-030b3200-9ec6-11ea-9538-4cb68ec334c8.png)